src/ape_socket.h

```diff
@@ -8,22 +8,32 @@
 #define __APE_SOCKET_H
 
 #include "ape_common.h"
+
 #include "ape_buffer.h"
 #include "ape_pool.h"
 #include "ape_lz4.h"
+#include "ape_dns.h"
 
 #ifdef _WIN32
-#define ioctl ioctlsocket
-#define hstrerror(x) ""
-#include "port/windows.h"
```

Comment:
The defines, has been moved in the .c file since they were not used anywhere else inside libapenetwork. But I am guessing that this is wrong and we want to keep these in the header file.